explorer/testdata/class/abstract_method_base_no_instance.carbon

```diff
@@ -0,0 +1,22 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// RUN: %{explorer-run}
+// RUN: %{explorer-run-trace}
+// CHECK:STDOUT: result: 0
```

Comment:
This shouldn't be result 0. See https://github.com/carbon-language/carbon-lang/pull/2528/files#r1071564430

Reply:
I believe this is OK, see reply :)